bundles/org.openhab.binding.solarman/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solarman"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="logger" extensible="true">
+		<label>Solarman Binding Thing</label>
```

Comment:
Can you adjust the labal and description to be meaningfull/represent the actual Thing. Maybe Logger ?